src/cmdstan/arguments/arg_seed.hpp

```diff
@@ -11,13 +11,19 @@ namespace cmdstan {
     arg_seed(): u_int_argument() {
       _name = "seed";
       _description = "Random number generator seed";
-      _validity = "seed > 0, if negative seed is generated from time";
+      _validity = "integer > 0 or -1 to generate seed from sys clock time";
```

Comment:
Rather than "sys clock time" could we just write that out? I don't know if casual users would understand `sys clock`... actually I googled it and couldn't find hits.

Maybe replace that with "system time"? If someone is confused and they search for it, the first hit I find is wiki: https://en.wikipedia.org/wiki/System_time; and I think that's correct enough.